src/Symfony/Component/Form/Tests/Command/DebugCommandTest.php

```diff
@@ -164,27 +164,29 @@ public function testDebugCustomFormTypeOption()
 Symfony\Component\Form\Tests\Command\FooType (foo)
 ==================================================
 
- ---------------- -----------------------------------------------------------%s
-  Required         true                                                      %w
- ---------------- -----------------------------------------------------------%s
-  Default          -                                                         %w
- ---------------- -----------------------------------------------------------%s
-  Allowed types    [                                                         %w
-                     "string"                                                %w
-                   ]                                                         %w
- ---------------- -----------------------------------------------------------%s
-  Allowed values   [                                                         %w
-                     "bar",                                                  %w
-                     "baz"                                                   %w
-                   ]                                                         %w
- ---------------- -----------------------------------------------------------%s
-  Normalizer       Closure(Options $options, $value) {                       %w
-                     class: "Symfony\Component\Form\Tests\Command\FooType"   %w
-                     this: Symfony\Component\Form\Tests\Command\FooType { …} %w
-                     file: "%s"%w
-                     line: "%d to %d"%w
-                   }                                                         %w
- ---------------- -----------------------------------------------------------%s
+ ---------------- -----------%s
+  Required         true      %s
+ ---------------- -----------%s
+  Default          -         %s
+ ---------------- -----------%s
+  Allowed types    [         %s
+                     "string"%s
+                   ]         %s
+ ---------------- -----------%s
+  Allowed values   [         %s
+                     "bar",  %s
+                     "baz"   %s
+                   ]         %s
+ ---------------- -----------%s
+  Normalizers      [         %s
+                     Closure(%s
+                       class:%s
```

Comment:
This change looks a bit weird. Is this really necessary?

Reply:
It is hard have to update everytime these fixtures with details (width, spaces, endlines, etc) I think by shorting the output would avoid bother in the future.

Please, let me know if this does not make sense or if I'm missing something important to test hiding the details of the closure, which is unreadable IMO.